protos/backend_service.proto

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+syntax = "proto3";
+
+package durabletask.protos.backend.v1;
+
+option csharp_namespace = "Microsoft.DurableTask.Protobuf";
+option java_package = "com.microsoft.durabletask.implementation.protobuf";
+option go_package = "/internal/protos";
```

Comment:
This should be a full URL, such as `github.com/microsoft/durabletask-protobuf/etc`

Reply:
Nice, I wasn't aware of this. I'll try it out in a subsequent commit.